akd_core/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
+name = "akd_core"
+version = "0.8.0"
+authors = ["Sean Lawlor <seanlawlor@fb.com>"]
+description = "Core utilities for the auditable-key-directory suite of crates (akd and akd_client)"
+license = "MIT OR Apache-2.0"
+edition = "2018"
+keywords = ["key-transparency", "akd"]
+repository = "https://github.com/novifinancial/akd"
+readme = "../README.md"
+# Uncomment for automated building of the protobuf Rust sources. Necessary if the .proto specs change
+build = "src/build.rs"
+
+[build-dependencies]
+protobuf-codegen = "3.2"
+protobuf-parse = "3.2"
+
+[features]
+# Disable all STD for the crate
+nostd = []
+# Default feature mix (blake3 + vrf verification logic)
+default = ["blake3/std", "vrf"]
```

Comment:
Does this mean `std` will always have to be imported when using `blake3`?

Reply:
So at it's core import, we do **not** bring in the `std` feature on `blake3`, so if you wanted nostd you would do something like

```toml
akd_client = { version = 0.8, default-features=false, features = ["blake3"] }
```

which would not select the `default` feature and therefore not bring in std through `blake3`. The syntax is a bit weird, I'll admit, and I'm looking at cleaning it up. It seems the more common pattern is to not have a `nostd` feature, but rather an `std` feature so the native functionality is `nostd`. I might (in a later PR) match that common pattern.